File: InputTestCode/Kotlin/KotlinBasics/4_Functions.kt
// Functions

fun typedFunction(a: Int, b: Int): Int {
    return a + b
}

fun callFunc(): Int {
    return typedFunction(1, 2)
}

fun functionBodyAsExpression(a: Int, b: Int) = a + b

fun voidReturnValueWithUnit(a: Int, b: Int): Unit {
    println("sum of $a and $b is ${a + b}")
}

fun voidReturnValue(a: Int, b: Int) {
	println("sum of $a and $b is ${a + b}")
}

File: SequalsKTranspiledCode/KotlinBasics_To_Swift/4_Functions.swift
// Functions

func typedFunction(a: Int, b: Int) -> Int {
    return a + b
}

func callFunc() -> Int {
    return typedFunction(a: 1, b: 2)
}

func functionBodyAsExpression(a: Int, b: Int) -> Int {
    a + b
}

func voidReturnValueWithUnit(a: Int, b: Int) {
    print("sum of \(a) and \(b) is \(a + b)")
}

func voidReturnValue(a: Int, b: Int) {
    print("sum of \(a) and \(b) is \(a + b)")
}